src/lib.rs

```diff
@@ -332,17 +362,17 @@ impl HexView {
     /// e.g. cursor (5, 0) will result in (6, 0) because of the 1 space spacing after the fourth char
     /// and cursor (9, 0) will result in (11, 0) because of the 1+1 spacing after the fourth and eighth char
     fn get_cursor_offset(&self) -> Vec2 {
-        self.cursor + get_cursor_offset(self.cursor)
+        self.cursor + get_cursor_offset(self.cursor, self.config.clone())
```

Comment:
here for example: this clone is unnecessary

Reply:
yes, removed as part of the previous change.